README.md

```diff
@@ -64,6 +64,13 @@ Example RESTful interaction:
 | Receive current test scenario                                         |  GET ``scenario``                                   |
 | Set test scenario  		                                             |  PUT ``scenario`` with arguments ``electricity_price=<'constant' or 'dynamic' or 'highly_dynamic'>``|
 
+## Development
+
+This repository uses pre-commit to ensure that the files meet standard formatting conventions (such as line spacing, layout, etc).
+Presently only a handful of checks are enabled and will expanded in the near future. To run pre-commit first install
+pre-commit using pip `pip install pre-commit`, then either install pre-commit into your GitHub checkout using
+`pre-commit install` or run manual on all files by calling `pre-commit run --all-files`.
```

Comment:
@nllong Are these two commands specific to a repository and have to be run within the root of a given repo (or where the pre-commit config file is?).  If so, I think add a few words to say where these commands need to be run.  Or are these commands global for all git repositories on my machine?

Reply:
Only the `pre-commit install` is specific to a repository whereas `pip install pre-commit` is specific to your install (and enabled) version of python. I'll update the text a bit to make it more clear. Also, users don't have to use this at all (although that would defeat the purpose of having formatters). Thanks!

Reply:
Ok I see.  Yes, just updating the text would be great.  Thanks!

Reply:
Okay, I just pushed updated text to this line. Regarding the line formatters: we haven't seen the issues with that across mac/windows/linux. Typically git manages the line endings and there are settings to change the default behavior. For example the image shows how it checks out as CRLF, but commits as LF.

![image](https://user-images.githubusercontent.com/1907354/102267255-df95ec00-3ed6-11eb-8a58-a7e78d57c1d0.png)

If issues arise we can commit a .gitattributes file that forces line endings to be just LF (which IMO is the correct default for this project).

Reply:
Great, thanks for the update and for the information!